i18n/es/docusaurus-plugin-content-docs/current/nodes/validate/add-a-validator.md

```diff
@@ -20,7 +20,7 @@ qué blockchains. Para agregar un validador, emitiremos
 [transacciones](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction)
 a la P-Chain.
 
-:::advertencia
+:::warning
 
```

Comment:
@laviniat1996 when using the translation bot always double check to make sure the [admonitions ](https://docusaurus.io/docs/markdown-features/admonitions) are in english